### PR TITLE
HEEDLS-252 Add completion summary fields to course content

### PR DESCRIPTION
**Pull request by benjimarshall**
## Changes
Add completion summary data to the `CourseContent` model, update the `GetCourseContent` method

## Testing
Amend unit tests for `GetCourseContent` to cover these new fields

### CODE REVIEW

File: DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs [hunk 1]
```diff
@@ -12,6 +12,11 @@ public class CourseContent
         public string? BannerText { get; }
         public bool IncludeCertification { get; }
         public DateTime? Completed { get; }
+        public int AssessAttempts { get; }
```
Comment (FridaTveit) [inline DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs:15]: This is specifically post learning assessment attempts right? If so, can we make that clear in the name? 🙂 

Oh and it's also actually a max assessment attemps right? So worth making that clear in the name as well 🙂

**Review by FridaTveit — COMMENTED** [review]
One comment about naming otherwise looks good 🙂

File: DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs (hunk 1 above)
Comment (benjimarshall) [reply DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs:15]: I think so, it's a field in `Applications`. Would something like `MaxPostLearningAssessmentAttempts` be good? 🙂

Comment (FridaTveit) [reply DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs:15]: Yes, that sounds good to me 🙂

Comment (benjimarshall) [reply DigitalLearningSolutions.Data/Models/CourseContent/CourseContent.cs]: Done in 86103de 🙂

**Review by FridaTveit — APPROVED** [review]
Looks good 🙂

**Pull request merged**